pkg/search/postgres/common.go

```diff
@@ -671,7 +671,7 @@ func RunGetManyQueryForSchema(ctx context.Context, schema *walker.Schema, q *v1.
 		return nil, err
 	}
 	if query == nil {
-		return nil, emptyQueryErr
+		return nil, nil
```

Comment:
Should we log error on dev builds? Getting a nil query itself is a correctness problem in calling code.

Reply:
Actually we do this in global search on purpose